cajita/src/Cajita_DynamicPartitioner.hpp

```diff
@@ -29,14 +29,15 @@ namespace Cajita
 {
 //---------------------------------------------------------------------------//
 /*!
-  Sparse mesh block partitioner. (Current Version: Support 3D only)
-  \tparam Device Kokkos device type.
-  \tparam CellPerTileDim Cells per tile per dimension.
-  \tparam NumSpaceDim Dimemsion (The current version support 3D only)
+  Dynamic mesh block partitioner. (Current Version: Support 3D only) There
+  should be no instantiation for this class without implementing any workload
+  computation. \tparam Device Kokkos device type. \tparam CellPerTileDim Cells
+  per tile per dimension. \tparam NumSpaceDim Dimemsion (The current version
+  support 3D only)
 */
 template <typename Device, unsigned long long CellPerTileDim = 4,
```

Comment:
I don't think the `CellPerTileDim` should exist outside the sparse derived version

Reply:
There is a class member called `_workload_per_tile` in the base class which the core partitioning algorithm is based on. It records the minimal dividable (unit) area when partitioning.

If we removed the `tile` concept, the whole partitioning algorithm existing in the base class need to be re-implement. My suggestion here is to enforce `CellPerTileDim=1` for the dense case.